docs/pages/guides/alternatives-for-expo-notifications.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Alternatives for Expo Notifications
+sidebar_title: Expo Notifications alternatives
+description: Learn about alternative push notification services to Expo Notifications.
+---
+
+import { BoxLink } from '~/ui/components/BoxLink';
+
+Expo simplifies push notification implementation by abstracting much of the complexity involved in communicating with [Firebase Cloud Messaging (FCM)](https://firebase.google.com/docs/cloud-messaging) and [Apple Push Notification Service (APNs)](https://developer.apple.com/documentation/usernotifications/setting-up-a-remote-notification-server). With Expo, you can handle notifications for both Android and iOS the same way, saving development time on both the front end and back end.
```

Comment:
One thing we need to make very clear is that while developing `expo-notifications`, we do not test it with any of these external providers.

We develop `expo-notifications` in tandem with our own push service (which uses FCM and APNs directly) and this tighter integration allows us to ship some advanced features cross-platform which normally are not cross-platform.

Simple push messages with title and body will work with any provider but advanced patterns may not. After all, some of these services have their own native and react native SDKs - e.g. https://documentation.onesignal.com/docs/react-native-sdk-setup